dart/lib/src/sentry_client.dart

```diff
@@ -399,13 +399,24 @@ class SentryClient {
   }
 
   SentryEvent _eventWithRemovedBreadcrumbsIfHandled(SentryEvent event) {
-    final mechanisms =
-        (event.exceptions ?? []).map((e) => e.mechanism).whereType<Mechanism>();
+    final mechanisms = (event.exceptions ?? [])
+        .map((e) => e.mechanism)
+        .whereType<Mechanism>();
     final hasNoMechanism = mechanisms.isEmpty;
-    final hasOnlyHandledMechanism =
-        mechanisms.every((e) => (e.handled ?? true));
 
-    if (hasNoMechanism || hasOnlyHandledMechanism) {
+    final hasOnlyHandledMechanism = mechanisms
+        .every((e) => (e.handled ?? true));
+
+    final unhandledButNotCrashingMechanisms = <String>[
+      'PlatformDispatcher.onError',
+      'isolateError',
+      'runZonedGuarded',
+    ];
```

Comment:
@denrase I don't think we need to special case the mechanism names, only the`handled` matters.

Reply:
k, i removed this again